docs/os/windows/windows-tools41.md

```diff
@@ -0,0 +1,292 @@
+
+ 5. **Windows 7 only:** Rename the private disk `D:` to `Q:`. This can be done by starting the diskmanager and changing the drive letter:
+	
+	 - Start the command prompt as administrator, i.e. right click on the Command Prompt icon (All Programs -> Accessories) and choose "Run as administrator"
+	 - In the command prompt type `diskmgmt.msc`
+	 - In the disk manager, select the volume `Private (D:)`
+	 - Select the option `Change Drive Letter and Path`
+	 - Select option `Change...`
+	 - Select the letter `Q`
+	 - Click `OK` in all still open windows of the disk manager and terminate it.
+
+ 6. Install Qubes Windows Tools by starting `qubes-tools-x64.msi` as administrator, optionally selecting the `Xen PV disk drivers`. For installation in a template, you should select `Move user profiles`. If during installation, the Xen driver requests a reboot, select "No" and let the installation continue - the system will be rebooted later.
+
+ 7. After successful installation, the Windows VM must be shut down and started again, possibly a couple of times. On each shutdown, wait until the VM is really stopped, i.e. Qubes shows no more activity.
+
+ 8. Qubes will automatically detect that the tools have been installed in the VM and will set appropriate properties for the VM, such as `qrexec_installed`, `guiagent_installed`, and `default_user`. This can be verified (but is not required) using the `qvm-prefs` command  *(where `<VMname>` is the name of your Windows VM)*:
+
+	        [user@dom0 ~] $ qvm-prefs <VMname>
+
+ 	It is advisable to set some other parameters in order to enable audio, synchronize the Windows clock with the Qubes clock, and so on:
+	        
+	        [user@dom0 ~] $ qvm-features <VMname> gui 1
+	        [user@dom0 ~] $ qvm-features <VMname> gui-emulated 1
+	        [user@dom0 ~] $ qvm-features <VMname> audio-model ich9
+	        [user@dom0 ~] $ qvm-features <VMname> stubdom-qrexec 1
+	        [user@dom0 ~] $ qvm-features <VMname> timezone localtime
+
+ 9. Reboot Windows. If the VM starts, but does not show any window then shutdown Windows from the Qube manager, wait until it has really stopped, and reboot Windows once more.
+ 
+ 10. Now the system should be up, with QWT running correctly.
+ 
+ 11. **Windows 7 only:** Optionally enable seamless mode on VM startup. This can be done by setting appropriate values in the Windows registry:
+	
+	 - Start the command prompt as administrator, i.e. right click on the Command Prompt icon (All Programs -> Accessories) and choose "Run as administrator"
+	 - In the command prompt type `regedit`
+	 - In the registry editor, position to the key `\HKEY_LOCAL_MACHINE\Software\Invisible Things Lab\Qubes Tools\`
+	 - Change the value `SeamlessMode` from 0 to 1
+	 - Position to the key `\HKEY_LOCAL_MACHINE\Software\Invisible Things Lab\Qubes Tools\qga\`
+	 - Change the value `SeamlessMode` from 0 to 1
+	 - Terminate the registry editor.
+	
+ 	 After the next boot, the VM will start in seamless mode.
```

Comment:
Might be useful remark that unfortunately it is not applied to AppVM based on TemplateVM. Such changes requires TemplateVM configuring.

Reply:
I didn't specify the unit for using a digital timezone value, because I did not find any documentation on this. Hopefully, someone does.